DigitalLearningSolutions.Web/Styles/trackingSystem/selectCourse.scss

```diff
@@ -1,26 +1,14 @@
-﻿@import "../shared/searchableElements/search";
-
-.course-dropdown-container {
-  display: flex;
-  flex-direction: row;
-  width: 100%;
-}
-
-.course-dropdown {
-  @extend .input-with-submit-button;
-  display: flex;
-  flex: 1;
-}
-
-.course-submit {
-  @extend .right-submit-button;
-
-  &__dropdown {
-    @extend .right-submit-button;
-    background-color: $button-dark-blue;
-
-    @include mq($from: tablet) {
-      width: 20%;
-    }
-  }
-}
+﻿@import "../shared/searchableElements/filter";
+
+.filter-dropdown {

```

Comment:
Do we want the Select Course dropdown to have the rounded corners too?

Reply:
Good idea, looks better like this
![localhost_5001_TrackingSystem_CourseSetup_AddCourse_SelectCourse (7)](https://user-images.githubusercontent.com/70278044/156182797-8f09803d-030d-42bb-b10e-54614bc43fa8.png)